webrecorder/browsertrix
# Manage org member roles and invites

Resolves https://github.com/webrecorder/browsertrix-cloud/issues/527:
- View and delete pending invites
- Update user roles for members
- Remove members

Follows https://github.com/webrecorder/browsertrix-cloud/pull/575

### Manual testing
Prerequisites: User that is an org owner/admin of an org with multiple users. Invite members to org to test with.
1. Log in as org owner/admin
2. Go to "Org Settings" -> "Members"
  a. If existing pending invites, verify "Pending Invites" are shown.
  b. If no existing pending invites, invite a new user and then verify.
3. Click trash icon next to pending invite. Verify invite is removed
4. Select user role for existing member. Verify user role updates as expected
5. Click remove icon next to existing member. Verify member is removed from org
6. Self removal:
  a. Verify remove button is disabled if current user is only admin, only in one org, or only org member.
  b. Add another admin member and log in as other admin. Verify window shows confirmation whether to delete self

### Screenshots
**Org Settings Members panel:**
<img width="911" alt="Screen Shot 2023-02-08 at 3 46 29 PM" src="https://user-images.githubusercontent.com/4672952/217677540-c57c4527-2976-4f48-bd5e-fae11cb1a46a.png">

**Self remove button enabled when multiple admins:**
<img width="924" alt="Screen Shot 2023-02-08 at 3 46 40 PM" src="https://user-images.githubusercontent.com/4672952/217677598-3ff995b8-5664-43c2-b621-521f924bc303.png">

**Confirmation when removing self:**
<img width="929" alt="Screen Shot 2023-02-08 at 3 46 47 PM" src="https://user-images.githubusercontent.com/4672952/217677615-1e05cb94-184f-4a54-9cc1-5e3294515729.png">
